src/js/components/campaignpage/Charities.jsx

```diff
@@ -58,6 +93,13 @@ const Charities = ({ charities, donation4charity }) => {
 	};
 
 	let sogiveCharitiesWithDonations = sogiveCharities.filter(c => getDonation(c)); // Get rid of charities with no logged donations.
+	if (hideCharities) {
```

Comment:
What data format is hideCharities? Maybe rename it to make it clearer.

Reply:
Added comments - it's represented by the KeySet prop control as an object of {charity id : bool}, which I then convert to an array